spec.html

```diff
@@ -33172,19 +33166,19 @@
 
       <!-- es6num="24.2.2.1" -->
       <emu-clause id="sec-dataview-buffer-byteoffset-bytelength">
-        <h1>DataView (_buffer_, _byteOffset_, _byteLength_ )</h1>
-        <p>`DataView` called with arguments _buffer_, _byteOffset_, and _byteLength_ performs the following steps:</p>
+        <h1>DataView (_buffer_ [ , _byteOffset_, _byteLength_ ] )</h1>
+        <p>When the `DataView` is called with at least one argument _buffer_, the following steps are taken:</p>
         <emu-alg>
           1. If NewTarget is *undefined*, throw a *TypeError* exception.
           1. If Type(_buffer_) is not Object, throw a *TypeError* exception.
           1. If _buffer_ does not have an [[ArrayBufferData]] internal slot, throw a *TypeError* exception.
-          1. Let _numberOffset_ be ? ToNumber(_byteOffset_).
-          1. Let _offset_ be ToInteger(_numberOffset_).
-          1. If _numberOffset_ &ne; _offset_ or _offset_ &lt; 0, throw a *RangeError* exception.
+          1. Let _offset_ be ? ToInteger(_byteOffset_).
+          1. If _offset_ &lt; 0, throw a *RangeError* exception.
+          1. If _offset_ is -0, let _offset_ be +0.
```

Comment:
:+1: Missed that one when I fixed the similar issue for the typed array constructors.